packages/api/src/types/common.d.ts

```diff
@@ -28,7 +36,7 @@ declare global {
       tokenId?: string;
 
       getBroadcasters?: () => Promise<NodeAddress[]>;
-      getOrchestrators?: () => Promise<NodeAddress[]>;
+      orchestratorsGetters?: Array<() => Promise<OrchestratorNodeAddress[]>>;
```

Comment:
Wait now I'm confused. I thought that the `getOrchestrators` function still existed on the request, doesn't it? Could we move that helper that foreaches on the getters and concatenates them all to the request, so we can re-use it anywhere we previously needed the orchs? Seems like we're doing `orchestratorGetters[0]` in a couple places (mostly tests for now?) that could be avoided if we still exposed the `getOrchestrators` helper as before, but now iterating over the `orchestratorGetters`. WDYT?